### PR TITLE
Fix: allow 0 count and 0 long_factor

### LINKED ISSUE #254: count cannot be zero
In `0.19` (since #220), the `~count` argument to `Test.make` can no longer be zero.

Why?

Is this documented?

Using a zero count can be used to disable a test. Real-life use:
https://github.com/andrenth/ocaml-stdint/blob/0.7.0/tests/stdint_test.ml#L6

### PR DESCRIPTION
**Pull request by vch9**
Closes #254

### CODE REVIEW

File: test/core/QCheck_unit_tests.ml [hunk 1]
```diff
@@ -179,11 +179,23 @@ module TestCount = struct
...
+    with
+    | _ -> ()
+
   let tests =
     ("Test.make ~count", Alcotest.[
          test_case "make with custom count" `Quick test_count_10;
          test_case "make with default count" `Quick test_count_default;
          test_case "make with env count" `Quick test_count_env;
+         test_case "make with 0 count" `Quick test_count_0;
+         test_case "make with negative count shoul fail"
```
Comment (jmid) [inline test/core/QCheck_unit_tests.ml:197]: (same typo)

**Comment by jmid** [comment]
Looks good to me - I just spotted a couple of typos.
Extra thanks for adding both QCheck and QCheck2 unit tests :pray:

File: test/core/QCheck2_unit_tests.ml [hunk 2]
```diff
@@ -179,11 +179,23 @@ module TestCount = struct
...
+    with
+    | _ -> ()
+
   let tests =
     ("Test.make ~count", Alcotest.[
          test_case "make with custom count" `Quick test_count_10;
          test_case "make with default count" `Quick test_count_default;
          test_case "make with env count" `Quick test_count_env;
+         test_case "make with 0 count" `Quick test_count_0;
+         test_case "make with negative count shoul fail"
```
[the remark this answers is not in the document]
Comment (vch9) [reply test/core/QCheck2_unit_tests.ml]: Thanks, fixed

File: test/core/QCheck_unit_tests.ml (hunk 1 above)
Comment (vch9) [reply test/core/QCheck_unit_tests.ml]: Thanks, fixed

**Pull request merged**